test/run-tests.sh

```diff
@@ -41,6 +41,12 @@ if [[ ! ( -x ../rgbasm && -x ../rgblink && -x ../rgbfix && -x ../rgbgfx ) ]]; th
 	false
 fi
 
+# We do not compile with `-fsanitize=leak` because macOS clang++ does not support it.
+# Instead, enable ASan (`-fsanitize=address`) to check for memory leaks.
+if [ "$(uname)" != "Darwin" ]; then
+	export ASAN_OPTIONS=detect_leaks=1
+fi
```

Comment:
Defining `__asan_default_options()` would let this work even outside of this test script... but the detection might be a lil' more annoying.

Reply:
Well, strictly speaking, I believe that this is overly broad, because x64 macOS would be supported. But GitHub's CI only provides AArch64 macOS 14 runners. Since that's part of our “private dev config”, being overly broad sounds fine to me.

Reply:
This is not test config, this is “debug build” config. This goes beyond just our test suite!